SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+Due to frequent breaking changes from the HoYoverse/Cognosphere regarding game repair indexes, cache files, update zip files, etc. we can only support any latest Collapse version, be it either from Stable or Preview branch.
+Additionally, due to major changes that happened on the Collapse, we cannot support any version below 1.69.0 and should be updated ASAP to latest version, either with in-build updater, or by reinstalling Collapse Launcher manually (if the built-in launcher fails to update from such old version of Collapse).
+| Version   | Supported          |
+| -------   | ------------------ |
+| >=1.69.0  | :white_check_mark: |
+| < 1.0.69  | :x:                |
+
+
+## Reporting a Vulnerability
+Report any vulnerability regarding Collapse Launcher here on Issues with the tag `[VULNERABILITY]` in the title. If its servere enough to not be safe to report it using GitHub issues, please contact @neon-nyan directly via Discord at `DreadCode#9025`.

```

Comment:
Rephrase: Please report any vulnerability pertaining to Collapse by creating an issue on GitHub, with **[VULNERABILITY]** in the issue title. If you believe that the issue is severe enough to warrant anonymity, please contact @neon-nyan (`DreadCode#9025`), @Cryotechnic (`Cry0#0002`) or @bagusnl (`bagusnl#2938`) directly on Discord.

Please allow us some time to process your report and get back to you in a timely manner. We are volunteers and maintain this project in our free time.

Reply:
Applied